src/L1CacheFactory.php

```diff
@@ -85,11 +104,9 @@ protected function createStatic($pool)
      */
     protected function createSQLite($pool)
     {
-        $hasApcu = function_exists('apcu_fetch');
-        // TODO: Maybe implement StateL1SQLite class instead of NULL one.
-        $state = $hasApcu ? new StateL1APCu("sqlite-$pool") : new StateL1Static();
-        $cache = new SQLiteL1($pool, $state);
-        return $cache;
+        $stateDriver = function_exists('apcu_fetch') ? 'apcu' : 'sqlite';
```

Comment:
Added APCu and SQLite as this are the ones that need to be done at some point. I am trusting the fall-back in the factory to instantiate static instead of SQLite state manager for now.